tg-portfolio/components/layyout/Navbar.jsx

```diff
@@ -48,24 +58,31 @@ const Navbar = () => {
             ))}
           </div>
         </div>
-        {/* Mobile Menu Button */}
-        <div className="md:hidden flex items-center">
-          button
+        {/* Mobile Menu */}
+        <div className="md:hidden flex items-center bg-green-800/90 w-full flex-row justify-between fixed">
+          <button onClick={toggleMenu} type="button"> <HiMenu className="h-8 w-8 text-white" /> </button>
+          <div style={{ display: open ? 'block' : 'none' }}>
+            <ul className="">
+              <div className="flex flex-auto flex-row mr-5 gap-1">
+                {navLinks.map((navItem) => (
+                  <div key={navItem.id}>
+                    <Link href={navItem.link}>
+                      <button type="button">
+                        <li
+                          key={navItem.listId}
```

Comment:
I don't think there is a need for this listId field. You can have a single unique ID per object and use that on the main element, i.e., over here, it is <Link />

Reply:
Thanks!!